src/form/index.html

```diff
@@ -32,7 +32,7 @@
 </head>
 <body>
 	<main>
-		<tp-form prevent-submit="yes">
+		<tp-form prevent-submit="yes" submit-reset="yes">
```

Comment:
@Takshil-Kunadia Let's rename to `reset-on-submit` - can you add a Loom of how this would work? Would it be a weird experience if the user suddenly saw all their values disappear? Also, would this conflict with the "submitting" feature?

Reply:
Hmmm.... this will run on submit event, not on "submit successful" which will reset the form before user is taken to thank you page.